test/onnx/onnx_import_dyn_shapes.in.cpp

```diff
@@ -550,3 +550,11 @@ NGRAPH_TEST(onnx_dyn_shapes_${BACKEND_NAME}, expand_uint16_dyn_shape)
 
     test_case.run();
 }
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, model_resize_opset10)
+{
+    const auto resize = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/resize_opset10.prototxt"));
+
+    auto test_case = ngraph::test::NgraphTestCase(resize, "${BACKEND_NAME}", BackendMode::DYNAMIC);
```

Comment:
This test is incomplete. Please provide it with inputs and outputs, if it's possible to run on any backend.
If not possible to run on any backends, let's remove it for now and add a story to add tests later.

Reply:
I added some ASSERTS on imported ngraph function instead.